build.sh

```diff
@@ -50,6 +50,9 @@ install_rpms() {
     # Process our base dependencies + build dependencies and install
     (echo "${builddeps}" && "${srcdir}"/src/print-dependencies.sh) | xargs yum -y install
 
+    # Freeze kernel due to https://github.com/coreos/coreos-assembler/issues/2003
+    yum -y install https://kojipkgs.fedoraproject.org//packages/kernel/5.9.16/200.fc33/"${arch}"/kernel-{core,modules}-5.9.16-200.fc33."${arch}".rpm
```

Comment:
just tested this and it fails because it still uses the new kernel and it's present:

```
[coreos-assembler]$ rpm -qa|grep -i kernel
kernel-core-5.10.7-200.fc33.s390x
kernel-headers-5.10.7-200.fc33.s390x
kernel-modules-5.10.7-200.fc33.s390x
kernel-core-5.9.16-200.fc33.s390x
kernel-modules-5.9.16-200.fc33.s390x
```
i had to remove the 5.10 ones to get it to work.

Reply:
Thanks, I updated this now and tested it locally!